src/hooks/transactions/useSignTransactions.tsx

```diff
@@ -162,13 +179,18 @@ export const useSignTransactions = () => {
     } catch (error) {
       const errorMessage =
         (error as Error)?.message || (error as string) || ERROR_SIGNING_TX;
+      console.error(errorMessage);
+
       dispatch(
         moveTransactionsToSignedState({
           sessionId,
           status: TransactionBatchStatusesEnum.cancelled
         })
       );
-      onCancel(errorMessage, sessionId);
+      onCancel(
+        errorMessage.includes('cancel') ? TRANSACTION_CANCELLED : errorMessage,
```

Comment:
soo, if the error message changes, this won't work?

Reply:
Right. I don't have control over what erdjs packages throw. I tried a general solution using what is currently thrown at this moment.